lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEF.cs

```diff
@@ -85,19 +90,56 @@ private static IConfiguration GetRuntimeConfiguration(string name)
                     return YARNClientConfiguration.ConfigurationModule.Build();
                 case YARNRest:
                     return YARNClientConfiguration.ConfigurationModuleYARNRest.Build();
+                case HDInsightLocal:
+                    {
+                        // To run against HDInsight please replace placeholders below, with actual values for
+                        // connection string, container name (available at Azure portal) and HDInsight 
+                        // credentials (username and password)
+                        const string connectionString = "ConnString";
+                        const string containerName = "foo";
+                        return
+                            new MultiRuntimeConfigurationBuilder().AddRuntime(RuntimeName.Yarn)
```

Comment:
I agree, but I don't think this is in the scope for this PR. If we want to improve HelloReef we can create a jira for that. The same goes for adding more environments not specifically multi runtime.